1/juliaanoonan/juliaanoonan1.py

```diff
@@ -0,0 +1,90 @@
+#Okay, let's do this
+
+import sys
+import math
+import numpy as np
+sys.path.insert(0, '../../extra-files')
+import masses
+
+def frequencies(mol,Hess):
+
+#3 build the mass-weighted Hessian (do simple example to convince yourself)
+
+    mas = np.array(np.repeat(mol.masses, 3) ** -0.5)
+    mas = np.diag(mas)
+    MwH = mas @ Hess @ mas
+
+#4 compute the eigenvalues and eigenvectors of the mass-weighted Hessian
+
+    k, q = np.linalg.eigh(MwH)
+
+#5 un-mass-weight eigenvectors to get normal coords (makes more sense on paper) remember the eigenvalues same
+
+    Q = mas @ q
+
+#6 determine the spatial frequencies in wavenumbers from force constants (eigenvalues)
+
+    htoJ = 4.359744e-18
```

Comment:
Sometimes I find it useful to cite where I got my conversion factors from. I think all of these conversion factors are correct